examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -6,11 +6,75 @@ using namespace chip::app::Clusters;
 
 void RvcDevice::Init()
 {
+//    mServiceAreaInstance.Init();
```

Comment:
Pretty sure this is my fault - for temporary compiling during the re-right.  Should be un-commented.

Reply:
Done.